deployment/cdk/opensearch-service-migration/lib/stack-composer.ts

```diff
@@ -144,10 +144,6 @@ export class StackComposer {
             throw new Error("Required context field 'contextId' not provided")
         }
         const contextJSON = this.parseContextBlock(scope, contextId)
-        console.log('Received following context block for deployment: ')
-        console.log(contextJSON)
-        console.log('End of context block.')
```

Comment:
Why are these being pulled out? They're pretty helpful for debugging

Reply:
When calling `cdk list` or any other commands these messages were being printed out to the same console output that was preventing parsing.

Would you to want me to add this back in? If so I'd like to check for a flag that prints the output rather than doing it by default

Reply:
This has been useful in the past for debugging. Is there a way we can turn this on only for deploy?

Reply:
Roger that, I'll look into this.

Reply:
I wasn't able to figure out a good 'switch' to print or not print this output, but instead I've added a deploy script, that allows the output to be printed and includes the other deployment defaults we use, example:

```
~/git/opensearch-migrations/deployment/cdk/opensearch-service-migration$ ./deploy.sh default
Context block for 'default':
---
{
   "stage": "dev",
   "targetCluster": {
      "endpoint": "http://TARGET_CLUSTER_ENDPOINT",
      "auth": {
         "type": "sigv4",
         "// basic auth documentation": "The next two lines are releavant for basic auth only",
         "username": "<USERNAME>",
         "passwordFromSecretArn": "<ARN_OF_SECRET_CONTAINING_PASSWORD>",
         "// sigv4 documentation": "The next two lines are releavant for sigv4 only",
         "region": "us-west-2",
         "serviceSigningName": "es"
      }
   },
   "sourceCluster": {
      "endpoint": "http://SOURCE_CLUSTER_ENDPOINT",
      "auth": {
         "type": "sigv4",
         "// sigv4 documentation": "The next two lines are releavant for sigv4 only",
         "region": "us-west-2",
         "serviceSigningName": "aoss"
      }
   },
   "reindexFromSnapshotServiceEnabled": true,
   "reindexFromSnapshotMaxShardSizeGiB": 80,
   "trafficReplayerServiceEnabled": false
}
---
```

Reply:
I like for users who are accustomed to cdk to have the familiarity with seeing us recommend commands like `cdk <command>` at the top level instead of behind a script. I too haven't found a normalized way to do this in CDK though. What are you thoughts around having a different log type for these informational messages that is on by default but can be switched off when provided with some context variable like `disableAnalysisOutput`?

Reply:
And you just approved - thanks!